frontend/src/views/CodingPage/CodingPageEasy.js

```diff
@@ -0,0 +1,121 @@
+import React, { useRef, useState, useEffect } from "react";
+import { Button } from "@mui/material";
+import "./codingPage.scss";
+import Editor from "components/Editor/Editor";
+import { io } from "socket.io-client";
+import { useUsername } from "slices/usernameSlice";
+import { useRoom } from "slices/roomSlice";
+
+import "codemirror/mode/javascript/javascript";
+import "codemirror/mode/python/python";
+
+function CodingPageEasy() {
+  const [language, setLanguage] = useState("python");
+  const [text, setText] = useState('print("hello world")');
+  const [question, setQuestion] = useState();
+  const [socket, setSocket] = useState(null);
+  const questionNumber = useRef(1);
+  const qnOne = useRef();
+  const qnTwo = useRef();
+
+  const readNewQuestion = async (firstQuestionHash, secondQuestionHash) => {
+    const firstQuestionNumber = (firstQuestionHash % 456) + 1;
```

Comment:
Some comments here:

- can set firstQuestionNumber and secondQuestion number to variable so it can be mutable if same
- just increment second question number by 1 and use modulo in case question number exceeds range

`    let firstQuestionNumber = (firstQuestionHash % 456) + 1;
    let secondQuestionNumber = (secondQuestionHash % 456) + 1;

    const questionOne = await import(
      `questions/easy/q${firstQuestionNumber}.js`
    );

    if (firstQuestionNumber === secondQuestionNumber) {
      secondQuestionNumber = (firstQuestionNumber + 1) % 456
    }

    const questionTwo = await import(`questions/easy/q${secondQuestionNumber}.js`);`

- same applies for medium and hard